local-cli/generator-android/templates/src/app/BUCK

```diff
@@ -0,0 +1,68 @@
+import re
+
+# See [Docs](https://buckbuild.com/) to learn what BUCK is.
```

Comment:
nit: to learn about Buck

Buck (the tool itself) is spelled Buck rather than BUCK.